aspnetcore/tutorials/grpc/grpc-start/samples/GrpcStart/GrpcGreeter.Server/Program.cs

```diff
@@ -16,10 +17,15 @@ public static void Main(string[] args)
         }
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
-            Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder =>
-                {
-                    webBuilder.UseStartup<Startup>();
-                });
+             Host.CreateDefaultBuilder(args)
+        .ConfigureWebHostDefaults(webBuilder =>
+        {
+            webBuilder.UseStartup<Startup>();
+            webBuilder.ConfigureKestrel((context, options) =>
+            {
+                options.Limits.MinRequestBodyDataRate = null;
```

Comment:
I'm not sure we'd want this to be part of the sample. Mainly because this is a temporary fix and we'll be removing the need in the next preview. I'd rather doc the required code for this preview release but remove it from the sample which runs fine without it anyway.